src/riak_cs_config.erl

```diff
@@ -277,16 +277,16 @@ set_md5_chunk_size(_) ->
 
 %% doc Check app.config to see if repl proxy_get is enabled
 %% Defaults to false.
+-spec proxy_get_active() -> boolean().
 proxy_get_active() ->
-    case application:get_env(riak_cs, proxy_get) of
-        {ok, enabled} ->
-            true;
-        {ok, disabled} ->
-            false;
-        {ok, _} ->
-            _ = lager:warning("proxy_get value in app.config is invalid"),
-            false;
-        undefined -> false
+    case application:get_env(riak_cs, proxy_get, false) of
+        enabled ->   true;
+        disabled -> false;
+        Flag when is_boolean(Flag) -> Flag;
+        Other ->
+            _ = lager:warning("proxy_get value in riak.conf (advanced.config) is invalid: ~p",
```

Comment:
No. I was wrong.
Cuttlefish uses schema and validate values. One can not set any value other than `boolean()`. If invalid value is set, advanced.config is probably wrong.